frontend/src/components/pages/topics/Tab.Messages/index.tsx

```diff
@@ -10,8 +10,8 @@
  */
 
 import { ClockCircleOutlined, DeleteOutlined, DownloadOutlined, SettingFilled, SettingOutlined } from '@ant-design/icons';
-import { DownloadIcon, PlusIcon, SkipIcon, SyncIcon, XCircleIcon, KebabHorizontalIcon } from '@primer/octicons-react';
-import { ConfigProvider, DatePicker, Radio, Select, Table } from 'antd';
+import { DownloadIcon, KebabHorizontalIcon, PlusIcon, SkipIcon, SyncIcon, XCircleIcon } from '@primer/octicons-react';
```

Comment:
In the future, before the migration ends, probably we should do an audit and consider using 1 set of icons like discussed previously. Of course nothing to change for now.

Reply:
@malinskibeniamin for sure, I'm 100% onboard with this, but hopefully in a different PR